.github/pull_request_template.md

```diff
@@ -0,0 +1,3 @@
+## What changed?
```

Comment:
**suggestion(non-blocking):** Include a reminder to link to the issue.  There might not always be one, but we do encourage folks to make their own issues!